mac/history.md

```diff
@@ -1,7 +1,7 @@
 # Keyman for macOS Version History
 
-## 2018-08-10 10.0.105 stable
-* Fixed bug in engine that caused incorrect rule to be used (#1091)
+## 2018-08-14 10.0.111 stable
```

Comment:
Don't we still want to leave version `10.0.105` in the history? Maybe use new verbage on what `111` fixes.

Reply:
I debated that. I wasn't sure. But if I do leave it, I probably need to change the wording to tell people not to use it.

Reply:
I'd actually prefer to just yank that version, but I don't know if we have a good way to do that.

Reply:
Or maybe I can list it as version 1.0.5 Unstable!